CashRegisterWithForms/shopping.js

```diff
@@ -0,0 +1,42 @@
+function countEver() {
+
+  	document.getElementById("show").style.display = "block";
+	
```

Comment:
let deliveryContainer = document.getElementById("show")
console.log(deliveryContainer)
{
 style: { 
 display: "block",
width: "100px",
....
},
value: {},
size: {}
placeholder: {}
}
let output = document....
output.value = "value with discount"/  "value without discount"/ "value with discount and delivery cost"